docs/src/developers_guide/converting_from_unittest.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../common_links.inc
+
+.. _converting_tests:
+
+*******************************************
+Converting From ``unittest`` to ``pytest``
+*******************************************
+
+Conversion Checklist
+--------------------
+.. note::
+    Please bear in mind the following checklist is for general use; there may be
+    some cases which require extra context or thought before implementing these changes.
+
+#. Before making any manual changes, run https://github.com/dannysepler/pytestify
+   on the file. This does a lot of the brunt work for you!
+#. Check for references to :class:`iris.tests.IrisTest`. If a class inherits
+   from this, remove it. :class:`iris.tests.IrisTest` has been deprecated, and
+   replaced with the :mod:`iris.tests._shared_utils` module.
+#. Check for references  to ``unittest``. Many of the functions within unittest
+   are also in pytest, so often you can just change where the function is imported
+   from.
+#. Check for references to `self.assert`. Pytest has a lighter-weight syntax for
+   assertions, e.g. ``assert x == 2`` instead of ``assertEqual(x, 2)``. In the
+   case of custom assertions, the majority of these have been replicated in
+   :mod:`iris.tests._shared_utils`, but with snake_case instead of camelCase.
+   Some :class:`iris.tests.IrisTest` assertions have not been converted into
```

Comment:
```suggestion
   case of custom :class:`~iris.tests.IrisTest` assertions, the majority of these
   have been replicated in
   :mod:`iris.tests._shared_utils`, but with snake_case instead of camelCase.
   Some custom assertions have not been converted into
```